scripts/cleanup/cleaner-vpc.py

```diff
@@ -33,24 +35,102 @@
 
 from datetime import datetime, timedelta
 import datetime
+import warnings
 
 api_key = os.getenv("RESOURCE_MANAGER_APIKEY")
 resource_group_name = os.getenv("RESOURCE_GROUP_NAME")
-
+user_prefix = os.getenv("RESOURCE_USER_PREFIX")
+vpc_routes = []
+vpc_route = {}
 # https://cloud.ibm.com/docs/vpc?topic=vpc-service-endpoints-for-vpc
 authenticator = IAMAuthenticator(api_key)
 service = VpcV1(authenticator=authenticator)
 service.set_service_url('https://us-east.iaas.cloud.ibm.com/v1')
 
-#  Listing VPCs
+#  Listing VPCs and routes
 print("List VPCs")
 try:
     vpcs = service.list_vpcs().get_result()['vpcs']
 except ApiException as e:
   print("List VPC failed with status code " + str(e.code) + ": " + e.message)
 
-for vpc in vpcs:
-    print(vpc['resource_group']['id'], "\t",  vpc['name'])
+for vpc in vpcs:    
+    print(vpc['resource_group']['id'], "\t",  vpc['name'], "\t" , vpc['default_security_group']['name'], "\t" ,vpc['crn'], "\t" , vpc['created_at'])
+    vpc_crn_id = vpc['crn'].split(":")[9]
+    rtables = service.list_vpc_routing_tables(vpc_crn_id).get_result()['routing_tables']
+    rtable_id = rtables[0]['id']
+    rt_routes = service.list_vpc_routing_table_routes(vpc_crn_id, rtable_id).get_result()['routes']
+    for route in rt_routes:
+        vpc_route['id'] = route['id']
+        vpc_route['name'] = route['name']
+        vpc_route['created_at'] = route['created_at'] 
+        vpc_routes.append(vpc_route)
```

Comment:
just to note, I don't think we'll need to 'manually' remove. Important for the audit.